### DISCUSSION TITLE
Vehicle Suspension Jittering (UNITY - using BEPUV1)

### DISCUSSION THREAD

**Discussion by TalhaEjaz03** [bepu/bepuphysics1#9] [Q&A]
Hi! 
I am trying to run the vehicle input (using vehicle class) from the demo in UNITY. The vehicle is constructed but it has some weird jittering when it sets on the ground. I haven't made any changes to the file. Regardless I have tried tweaking values to no result.

I was successful in implementing the vehicle made from the suspension demo, it also had the same weird jittering issues but i resolved them by removing the spring suspension joints and made it rigid. My requirement for the vehicle don't require any suspension, but can nonetheless use it if its not wacky!!

Any help will be greatly appreciated !!

**Comment by RossNordby** [comment]
I'll need more information to narrow things down.

Can you reproduce the issue in the demos?

What timestep duration are you using in the simulation?
